File: Day-19/Prime.js
let n = 14;
let bool = true;
for (let i = 2; i < n; i++) {
  if (n % i == 0) {
    bool = false;
    break;
  }
}

if (bool) {
  console.log("Is a Prime");
} else {
  console.log("Not a prime");
}

File: Day-19/Prime.py
n = int(input())

i = 2
isPrime = True
while (i<n):
    if(n%i == 0):
        isPrime = False
        break;
    
    i += 1

if(isPrime):
    print('It is a prime number')
else:
    print('It is not a prime number')